### PR TITLE
Include compatible headers for C++

### PR DESCRIPTION
**Pull request by daemyung**

### CODE REVIEW

File: libs/filamat/include/filamat/MaterialBuilder.h [hunk 1]
```diff
@@ -39,8 +39,8 @@
 #include <utility>
 #include <vector>
 
-#include <stddef.h>
-#include <stdint.h>
+#include <cstddef>
+#include <cstdint>
```
Comment (pixelflinger) [inline libs/filamat/include/filamat/MaterialBuilder.h:43]: We should not use STL include in public headers. Is this fixing an issue?

**Review by pixelflinger — CHANGES_REQUESTED** [review]

File: libs/filamat/include/filamat/MaterialBuilder.h (hunk 1 above)
Comment (pixelflinger) [reply libs/filamat/include/filamat/MaterialBuilder.h:43]: I realize this header has tons of STL headers, but I think that's not ideal. Anyways, let's not add more if it's not needed.

Comment (daemyung) [reply libs/filamat/include/filamat/MaterialBuilder.h:43]: > We should not use STL include in public headers. Is this fixing an issue?

No it's not fixing any issues. For C++ project, including compatible headers is the canonical way. Including C headers isn't recommended. So I replaced including headers. If including C headers is the coding style of filament, please let me know. I will close this PR.

Comment (pixelflinger) [reply libs/filamat/include/filamat/MaterialBuilder.h:43]: yes it's the coding style we are using so far. If we ever change it, we would update all headers at once.

Comment (daemyung) [reply libs/filamat/include/filamat/MaterialBuilder.h:43]: Understood I will close this PR. Thanks.

**Pull request closed without merging**